e2e/jenkins/crw-ci/pr-check/k8s/Jenkinsfile

```diff
@@ -14,225 +14,297 @@ pipeline {
         PATH = "/tmp:/qa/tools/opt/apache-maven-3.5.4/bin:$PATH"
         JENKINS_BUILD = "true"
 
-        DEVFILE_URL = "${WORKSPACE}/e2e/files/happy-path/happy-path-workspace.yaml"
         SUCCESS_THRESHOLD = 5
 
-        CHE_IMAGE_REPO = "maxura/che-server"
-        CHE_IMAGE_TAG = "${ghprbPullId}"
-
         LOGS_AND_CONFIGS="${WORKSPACE}/logs-and-configs"
     }
 
-    stages {
-        stage("Cleanup") {
-            steps {
-                script() {
-                    sh "/usr/local/bin/minikube delete || true"
-                }
-            }
-        }
+    parameters {
+        string(name: 'cheImageRepo',
+                defaultValue: "maxura/che-server",
+                description: 'Repo of Che server image')
 
+        string(name: 'cheImageTag',
+                defaultValue: "$ghprbPullId",
+                description: 'Tag of Che server image')
+
+        booleanParam(name: 'buildChe',
+                defaultValue: true,
+                description: 'Should build Che server')
+
+        string(name: 'e2eTestToRun',
+                defaultValue: "test-happy-path",
+                description: 'TypeScript E2E test to run')
+        
+        string(name: 'testWorkspaceDevfileUrl',
+                defaultValue: "https://raw.githubusercontent.com/eclipse/che/${ghprbSourceBranch}/e2e/files/happy-path/happy-path-workspace.yaml",
+                description: 'URL of devfile of test workspace')
+    }
+
+    stages {
         stage("Prepare to start Che on K8S") {
             failFast true
 
             parallel {
                 stage("Download chectl") {
                     steps {
-                        script {
-                            // TO-DO use option "--install-path" https://github.com/eclipse/che/pull/14182
-                            sh """
-                               curl -sL  https://www.eclipse.org/che/chectl/ > install_chectl.sh
-                               chmod +x install_chectl.sh
-                               sudo PATH=$PATH ./install_chectl.sh --channel=next
-                               sudo mv /usr/local/bin/chectl ${WORKSPACE}/chectl
-                               sudo chmod +x ${WORKSPACE}/chectl
-                            """
-                        }
+                        // TO-DO use option "--install-path" https://github.com/eclipse/che/pull/14182
```

Comment:
We already have a solution for downloading just chectl binary (without the whole installation script & isntalling into global locations).
This note is not blocking PR merging (mostly because we are using one use slaves)